pkg/config/config_template.yaml

```diff
@@ -1912,22 +1912,35 @@ api_key:
   #
   # fim_enabled: false
 
+{{- if (eq .OS "windows")}}
+  ## @param sysprobe_socket - string - optional - default: localhost:3334
+  ## @env DD_SYSTEM_PROBE_CONFIG_SYSPROBE_SOCKET - string - optional - default: localhost:3334
+  ## The TCP address where the security runtime module is accessed.
+  #
+  # socket: localhost:3334
```

Comment:
The `@param` above is `sysprobe_socket`, which doesn't match here or the usages in `pkg/config/system_probe_cws_windows.go`. Which is correct?

Reply:
sysprobe socket & socket are different; socket is for the cws component.
However, I have a note (for a separate pr) that the system probe config needs some windows-specific updating.